### DISCUSSION TITLE
websocket support

### DISCUSSION THREAD

**Discussion by brucelane** [vcync/modV#513] [Ideas]
are you open to a PR for websocket support?

**Comment by 2xAA** [comment]
Hey Bruce,

How would you use the websockets?

**Comment by brucelane** [comment]
Hi Sam,
To exchange ui control values or groups between software, but might be too much work for a useful result.
I thought there were resulting shaders to send/receive from other software, but as it is textures it's less relevant then...
Anyway, I'm new here, I have to study modV source before...

**Reply by 2xAA** [reply]
modV could definitely receive shaders, we may need to write an additional API for that but it’s possible.

**Comment by shreeswifty** [comment]
Hi Bruce!

On Sat, Dec 19, 2020 at 10:50 AM Bruce LANE <notifications@github.com>
wrote:

> Hi Sam,
> To exchange ui control values or groups between software, but might be too
> much work for a useful result.
> I thought there were resulting shaders to send/receive from other
> software, but as it is textures it's less relevant then...
> Anyway, I'm new here, I have to study modV source before...
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/vcync/modV/discussions/513#discussioncomment-225400>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAGJY6XD6OBG2AGIT56MPFDSVTDVZANCNFSM4U7GCGTQ>
> .
>
-- 
Patrick Pagano B.S.,M.F.A
Assistant Professor in Residence
Digital Media Design & Development
University of Connecticut, Stamford

**Reply by brucelane** [reply]
Hi Patrick, long time no talk!

**Comment by shreeswifty** [comment]
i hope you are well!
funny meeting you here :-)


On Sat, Dec 19, 2020 at 1:01 PM Bruce LANE <notifications@github.com> wrote:

> Hi Patrick, long time no talk!
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/vcync/modV/discussions/513#discussioncomment-225708>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAGJY6RXVPB4ETOGVHQDQ7TSVTS5XANCNFSM4U7GCGTQ>
> .
>


-- 
*Patrick Pagano  B.S, M.F.A*
*Assistant Professor, *Interactive Media & Education
Audio Projection & Performance Design
Howard Community College
*Columbia, Maryland USA*
*http://shreeswifty.github.io <http://shreeswifty.github.io>*
*(352)226-2016*
